test/automation/src/sql/notebook.ts

```diff
@@ -121,6 +121,7 @@ export class Notebook {
 	}
 
 	// Code Cell Actions
+	private static readonly codeCellSelector = 'code-component div.editor';
```

Comment:
We don't need the changes in this file anymore right?

Reply:
oh that's right let me remove this